OpenKh.Tools.Kh2TextEditor/ViewModels/TextEditorViewModel.cs

```diff
@@ -127,6 +139,66 @@ public TextEditorViewModel()
             Drawing = new SpriteDrawingDirect3D();
             CurrentMessageEncoder = Encoders.InternationalSystem;
             _messages = new MessagesModel(this, this, new Msg.Entry[] { });
+
+            HandleReturn = new RelayCommand(x =>
+            {
+                Text += "{:newline}";                
+                SelectedItem = SelectedItem;
+            });
+
+            HandleRemoval = new RelayCommand(x =>
+            {
+                MessageEntries.RemoveAll(x => x.Id == SelectedItem.Id);
+                ResetMessagesView();
+            });
+
+            HandleAddition = new RelayCommand(x =>
+            {
+                int _id = MessageEntries[MessageEntries.Count - 1].Id;
+                MessageEntries.Add(new Msg.Entry() { Id = _id + 1, Data = new byte[] { 0x33, 0x2E, 0x38, 0x32, 0x00 } });
```

Comment:
Here you are assuming that the sequence `0x33, 0x2E, 0x38, 0x32, 0x00` is always `FAKE` regardless the region. I suggest to use `CurrentMessageEncoder.Encode(MsgSerializer.DeserializeText("FAKE").ToList())` to get the byte array. In that way you will be always sure to use the bytes for that specific region encoding. Sorry if it looks a bit verbose.

Reply:
"FAKE" is region independent. That is why. Capital letters currently exist in all regions. Would change if necessary but, I could implement your change regardless.